Source/Hurl.Selector/Pages/SelectorPage.xaml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Page
+    x:Class="Hurl.Selector.Pages.SelectorPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="using:System.Windows.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:Hurl.Selector.Pages"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:models="using:Hurl.Selector.Models"
+    mc:Ignorable="d">
+
+    <Grid Padding="8" RowSpacing="8">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+
+        <Grid>
+            <TextBlock
+                Padding="4,0"
+                Style="{StaticResource BodyTextBlockStyle}"
+                Text="Select a browser" />
+        </Grid>
+
+        <ItemsRepeater
+            Grid.Row="1"
+            VerticalAlignment="Center"
+            ItemsSource="{x:Bind ViewModel.Browsers, Mode=OneWay}">
+            <ItemsRepeater.Layout>
+                <UniformGridLayout
+                    ItemsJustification="Center"
+                    MaximumRowsOrColumns="4"
+                    MinColumnSpacing="12"
+                    MinItemHeight="120"
+                    MinItemWidth="120"
+                    MinRowSpacing="8" />
+            </ItemsRepeater.Layout>
+
+            <ItemsRepeater.ItemTemplate>
+                <DataTemplate x:DataType="models:Browser">
+                    <Grid>
+                        <Button
+                            Padding="8"
+                            HorizontalAlignment="Stretch"
+                            Command="{Binding ViewModel.LaunchBrowserCommand}"
+                            CommandParameter="{x:Bind}">
+                            <Grid RowSpacing="8">
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="78" />
+                                    <RowDefinition />
+                                </Grid.RowDefinitions>
+                                <Image Source="{x:Bind Icon, Mode=OneWay}" />
+                                <TextBlock
+                                    Grid.Row="1"
+                                    FontSize="10"
```

Comment:
Way too small. Try to stick with guidelines (see type ramp)

Reply:
set to 12 now